### ISSUE TITLE
MacOs M1 compatibility

### ISSUE THREAD

**Issue by andreakiro** [seemoo-lab/opendrop#81]
Hi all! I tried to install and use `opendrop` but on my first try I got the `AttributeError` below, which I've never seen before. I have the latest Apple M1 chip, and I've already had a lot of problems when installing and using small libraries. I was wondering if this specific error is related to M1 or if anyone else has had something similar? Thanks!

```
AttributeError: dlsym(RTLD_DEFAULT, archive_version_number): symbol not found
```

**Comment by megatunger** [comment]
Had you installed libarchive?

**Comment by andreakiro** [comment]
Yes. I installed libarchive via Homebrew, then followed step by steps their installation process.
